lib/_tokenUtils.ts

```diff
@@ -26,7 +26,12 @@ export const getCookie = (name: string): string => {
   return "";
 };
 
-export function setAnonymousUserToken(): void {
+export function setAnonymousUserToken(inMemory = false): void {
+  if (inMemory) {
+    this.setUserToken(`anon-${createUUID()}`);
```

Comment:
Used the `anon-` prefix here to match with the implementation in v3.

Reply:
even though we already use the anonymous- prefix for cookie-stored tokens? Should we be able to distinguish between the two? if so maybe the prefix should be different?

Reply:
I think from an analytics perspective it doesn't matter what prefix is used, as long as the token itself stays the same during is lifecycle. I just went with what's in v3, but I also initially wanted to name it `transient-xxx` so I'm not really locking in on this.

An advantage of having a different prefix than the cookie-based token is that it makes it easier to identify the type of user token in logs.